### PR TITLE
[openthermgateway] Add Diagnostics indication channel

### PR DESCRIPTION
**Pull request by ArjenKorevaar**
Signed-off-by: Arjen Korevaar <a.korevaar@mephix.com>

Add Diagnostics indication channel

See #9515

### CODE REVIEW

File: bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/OpenThermGatewayBindingConstants.java [hunk 1]
```diff
@@ -80,5 +81,6 @@
                     CHANNEL_DOMESTIC_HOT_WATER_SETPOINT, CHANNEL_FLAME, CHANNEL_RELATIVE_MODULATION_LEVEL,
                     CHANNEL_MAXIMUM_MODULATION_LEVEL, CHANNEL_FAULT, CHANNEL_SERVICEREQUEST, CHANNEL_REMOTE_RESET,
                     CHANNEL_LOW_WATER_PRESSURE, CHANNEL_GAS_FLAME_FAULT, CHANNEL_AIR_PRESSURE_FAULT,
-                    CHANNEL_WATER_OVER_TEMP, CHANNEL_OEM_FAULTCODE).collect(Collectors.toSet()));
+                    CHANNEL_WATER_OVER_TEMP, CHANNEL_OEM_FAULTCODE, CHANNEL_DIAGNOSTICS_INDICATION)
+            .collect(Collectors.toSet()));
```
Comment (J-N-K) [inline bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/OpenThermGatewayBindingConstants.java:85]:
```suggestion
           ;
```

File: bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/OpenThermGatewayBindingConstants.java [hunk 2]
```diff
@@ -68,10 +68,11 @@
     public static final String CHANNEL_AIR_PRESSURE_FAULT = "airpressfault";
     public static final String CHANNEL_WATER_OVER_TEMP = "waterovtemp";
     public static final String CHANNEL_OEM_FAULTCODE = "oemfaultcode";
+    public static final String CHANNEL_DIAGNOSTICS_INDICATION = "diag";
 
-    public static final Set<String> SUPPORTED_CHANNEL_IDS = Collections
-            .unmodifiableSet(Stream.of(CHANNEL_ROOM_TEMPERATURE, CHANNEL_ROOM_SETPOINT, CHANNEL_FLOW_TEMPERATURE,
-                    CHANNEL_RETURN_TEMPERATURE, CHANNEL_OUTSIDE_TEMPERATURE, CHANNEL_CENTRAL_HEATING_WATER_PRESSURE,
+    public static final Set<String> SUPPORTED_CHANNEL_IDS = Collections.unmodifiableSet(Stream
+            .of(CHANNEL_ROOM_TEMPERATURE, CHANNEL_ROOM_SETPOINT, CHANNEL_FLOW_TEMPERATURE, CHANNEL_RETURN_TEMPERATURE,
```
Comment (J-N-K) [inline bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/OpenThermGatewayBindingConstants.java:74]:
```suggestion
    public static final Set<String> SUPPORTED_CHANNEL_IDS = Set
            .of(CHANNEL_ROOM_TEMPERATURE, CHANNEL_ROOM_SETPOINT, CHANNEL_FLOW_TEMPERATURE, CHANNEL_RETURN_TEMPERATURE,
```

**Review by J-N-K — CHANGES_REQUESTED** [review]
Is this working without any code-changes?

**Comment by ArjenKorevaar** [comment]
> Is this working without any code-changes?

Yes, DataItemGroup.java contains a complete list of possible opentherm commands, based on the opentherm 2.2 specifications. So in this case adding a new channel to bind to the opentherm command was all that's needed.

**Review by J-N-K — APPROVED** [review]

**Comment by J-N-K** [comment]
Jenkins failed unrelated.

**Pull request merged**